lib/tasks/pacbio_data.rake

```diff
@@ -15,54 +15,65 @@ namespace :pacbio_data do
     puts '-> Pacbio plates successfully created'
 
     puts '-> Creating pacbio libraries...'
```

Comment:
This was more work than I expected. But the pools created by this task were all quite similar in composition, so this adds a bit of variety to make testing easier.